diff --git a/android/build.gradle b/android/build.gradle
index a28f0dd49603fefe038e4e7a913f9930bb0a6b18..59d8ab5e096097eefc60728514563f49268605f6 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -31,6 +31,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
   
\ No newline at end of file
